manifests/cpp.yml

```diff
@@ -180,8 +180,7 @@ tests/:
       Test_TelemetrySCAEnvVar: missing_feature
     test_tracer.py:
       Test_TracerSCITagging: missing_feature
-    test_tracer_flare.py:
-      TestTracerFlareV1: missing_feature
+    test_tracer_flare.py: missing_feature
```

Comment:
Is this change  on purpose included in this PR ?

Reply:
yes. Always been a false positive.

Reply:
Currently there is only `TestTracerFlareV1` in `test_tracer_flare.py`, so you it won't change anything on the test outcome. But it will skip by default any future classes that may be added in this file in the future. Is it what you want ?

Reply:
It is indeed the desired outcome. The file is called `test_tracer_flare.py`, it's fair to assume any future changes will test further the tracer flares feature, which is not implemented yet.